libraries/AP_DDS/AP_DDS_Client.h

```diff
@@ -28,6 +28,8 @@
 #define DDS_MTU             512
 #define DDS_STREAM_HISTORY  8
 #define DDS_BUFFER_SIZE     DDS_MTU * DDS_STREAM_HISTORY
+#define debug(fmt, args ...) do { if (debug_level) { GCS_SEND_TEXT(MAV_SEVERITY_INFO, "DDS_DEBUG: " fmt, ## args); } } while (0)
```

Comment:
why not pass a debug level to the debug() macro? so:
  debug(3, MAV_SEVERITY_INFO, "DDS: an unimportant debug message");

Reply:
Could potentially just use the MAV_SEVERITY_LEVEL.... `MAV_SEVERITY_INFO` in this case.  Do you need the extra parameter?